app/controllers/questions_controller.rb

```diff
@@ -36,12 +37,14 @@ def create
     answers_param.each.with_index(1) do |answer, index|
       @question.answers.build(:content => answer, :correct => false)
     end
+    @question.published_at = Time.zone.now if publishing?
 
     respond_to do |format|
       if @question.save
+        @question.published_at = Time.zone.now if publishing?
         format.html { redirect_to @question, notice: 'Question was successfully created.' }
         format.json { render :show, status: :created, location: @question }
-      else
+      else 
```

Comment:
Get rid of this trailing whitespace. In the future, you can avoid this by making sure your editor is configured to trim trailing whitespace (or respect the `.editorconfig`).